src/main/java/me/damonkelley/ui/GamePresenter.java

```diff
@@ -2,63 +2,90 @@
 
 import me.damonkelley.tictactoe.Game;
 import me.damonkelley.tictactoe.Marker;
-import me.damonkelley.tictactoe.Space;
+
+import java.util.ArrayList;
 
 class GamePresenter {
-    private StringBuffer output = new StringBuffer();
     private Game game;
     private int id = 1;
 
+    private final String THREE_BY_THREE = " %s | %s | %s \n" +
+                                          "---+---+---\n" +
+                                          " %s | %s | %s \n" +
+                                          "---+---+---\n" +
+                                          " %s | %s | %s \n" +
+                                          "\n";
+
+   private final String FOUR_BY_FOUR =    " %s | %s | %s | %s \n" +
+                                          "-----+-----+-----+-----\n" +
+                                          " %s | %s | %s | %s \n"  +
+                                          "-----+-----+-----+-----\n" +
+                                          " %s | %s | %s | %s \n"  +
+                                          "-----+-----+-----+-----\n" +
+                                          " %s | %s | %s | %s \n"  +
+                                          "\n";
+
     public GamePresenter(Game game) {
         this.game = game;
```

Comment:
As far as I can tell, this class only uses the `Game` to get ahold of a `Board`. So you end up with some unneeded dependencies - a little extra knowledge that you could eliminate by only taking in what you need. See also [Law of Demeter](https://en.wikipedia.org/wiki/Law_of_Demeter).